### PR TITLE
Migrator for linking models

### LINKED ISSUE #191: Kapselung: Migrator

### PR DESCRIPTION
**Pull request by wehrlean**
[Trello Card](https://trello.com/c/PAZWm3jo/701-kapselung-migrator-schreiben)

### CODE REVIEW

File: bundles/specmate-dbprovider-api/src/com/specmate/dbprovider/api/migration/IAttributeToSQLMapper.java [hunk 1]
```diff
@@ -89,6 +89,15 @@ void migrateNewDoubleAttribute(String objectName, String attributeName, Double d
 	 * @throws SpecmateException
 	 */
 	void migrateNewObjectReference(String objectName, String attributeName) throws SpecmateException;
```
Comment (junkerm) [inline bundles/specmate-dbprovider-api/src/com/specmate/dbprovider/api/migration/IAttributeToSQLMapper.java]: what multiplicity is this migrationcode doing? please change the method name accordingly to be consistent

File: bundles/specmate-dbprovider-h2/src/specmate/dbprovider/h2/AttributeToSQLMapper.java [hunk 2]
```diff
@@ -103,6 +103,16 @@ public void migrateNewObjectReference(String objectName, String attributeName) t
 		migrateNewReference(objectName, attributeName, "BIGINT");
 	}
 
+	@Override
+	public void migrateNewObjectReferenceOneToN(String objectName, String attributeName) throws SpecmateException {
```
Comment (junkerm) [inline bundles/specmate-dbprovider-h2/src/specmate/dbprovider/h2/AttributeToSQLMapper.java]: please add this to the oracle version as well

**Review by junkerm — CHANGES_REQUESTED** [review]
Please also add tests to the migration test package

**Comment by junkerm** [comment]
@wehrlean I fixed the issues and added tests - could you do a review?

**Comment by wehrlean** [comment]
It works perfect on my computer. Tests pass local. Travis is at the moment not available, but I think we can merge it.

**Comment by wehrlean** [comment]
@junkerm Kann man mergen, oder?

**Pull request merged**